src/app/datastore/entities/pokemon-detail.model.ts

```diff
@@ -1,35 +1,65 @@
+import { PokemonStatConstructor, PokemonStatModel} from "./pokemon-stats.model";
 import { PokemonTypeConstructor, PokemonTypeModel } from "./pokemon-type.model";
 
 export interface PokemonDetailConstructor {
   id: number;
+  sprites: {other: {"official-artwork": {front_default: string}}}; 
   height: number;
   weight: number;
   order: number;
+  stats: PokemonStatConstructor[];
   species: {name: string};
   types: PokemonTypes[];
 }
 
 interface PokemonTypes {
-    type: PokemonTypeConstructor;
+  type: PokemonTypeConstructor;
 }
 
 export class PokemonDetailModel {
   public id: number;
+  public sprites: string;
```

Comment:
change the name of the variable with something more meaningful. 
Just because in the API it's called that way it doesn't mean ours variable should be called that too.